File: ios/DrawHelperManager.m
//
//  HelperManagerBridge.m
//  RNViewRecorder
//
//  Created by Toddle on 12/01/2020.
//  Copyright © 2020 Facebook. All rights reserved.
//

#import <Foundation/Foundation.h>

// CalendarManagerBridge.m
#import "React/RCTViewManager.h"

@interface RCT_EXTERN_MODULE(DrawHelperManager, NSObject)

RCT_EXTERN_METHOD(isDeviceCompatible:(RCTResponseSenderBlock)callback)

@end

File: ios/DrawHelperManager.swift
//
//  HelperManager.swift
//  RNViewRecorder
//
//  Created by Toddle on 12/01/2020.
//  Copyright © 2020 Facebook. All rights reserved.
//

import Foundation
import UIKit

// CalendarManager.swift

@objc(DrawHelperManager)
class DrawHelperManager: NSObject {
  
  @objc func isDeviceCompatible(_ callback: (NSObject) -> ()) {
    let context = CIContext()
    
    
    callback( [[
        "isCompatible": context.description.contains("opengl")
        ]] as NSObject)
  }
  
}
